src/app/shared/components/header/header.component.html

```diff
@@ -1,15 +1,17 @@
 <mat-toolbar class="c-header">
   <mat-toolbar-row>
-    <sofie-logo [text]="title" (click)="navigateHome()"></sofie-logo>
+    <a class="logo-link" [routerLink]="[Paths.HOME]">
+      <sofie-logo [text]="title"></sofie-logo>
+    </a>
   </mat-toolbar-row>
 
   <mat-toolbar-row class="c-header__main-menu">
     <mat-nav-list>
-      <a class="c-header__nav-item" (click)="navigateToRundown()" i18n>header-component.rundowns.label</a>
-      <a class="c-header__nav-item" (click)="navigateToStatus()" i18n>header-component.status.label</a>
+      <a class="c-header__nav-item" [routerLink]="['/', Paths.RUNDOWNS]" i18n>header-component.rundowns.label</a>
+      <a class="c-header__nav-item" [routerLink]="['/', Paths.STATUS]" i18n>header-component.status.label</a>
```

Comment:
I can see that in the settings page, we use `routerLinkActive` to denote relative paths (if I am correct).
If we decide to use that for relative navigations, then `'/', ` is not necessary for the `routerLink` for absolute paths. I might be wrong though :smile:

Reply:
@KvelaGorrrrnio Maybe I miss something but I don't see relation between routerLinkActive and route relativity. The only way to told Angular that route is absolute, not relative is to start with `"/"`. I can add routerLinkActive logic to follow the active route if we want. If I miss something I stay tuned to add it. 

Side Node: Rundown overview links are added too.

Reply:
You are correct. There is no relation between them.